EventGenerator/src/SingleProcessGenerator_module.cc

```diff
@@ -123,14 +134,13 @@ namespace mu2e {
     auto daughters = gen->generate();
     for(const auto& d: daughters) {
 
-      output->emplace_back(mustop,
-                           d.creationCode,
-                           d.pdgId,
-                           mustop->endPosition(),
-                           d.fourmom,
-                           time
-                           );
-
+        output->emplace_back(mustop,
+                             d.creationCode,
+                             d.pdgId,
+                             mustop->endPosition(),
+                             d.fourmom,
+                             time
+                             );
```

Comment:
these look like whitespace changes.  Please make sure the indentation isn't changed.